9.2/root/usr/share/container-scripts/postgresql/common.sh

```diff
@@ -93,13 +93,22 @@ function generate_postgresql_recovery_config() {
 
 # Generate passwd file based on current uid
 function generate_passwd_file() {
-  export USER_ID=$(id -u)
-  export GROUP_ID=$(id -g)
-  grep -v ^postgres /etc/passwd > "$HOME/passwd"
-  echo "postgres:x:${USER_ID}:${GROUP_ID}:PostgreSQL Server:${HOME}:/bin/bash" >> "$HOME/passwd"
-  export LD_PRELOAD=libnss_wrapper.so
-  export NSS_WRAPPER_PASSWD=${HOME}/passwd
-  export NSS_WRAPPER_GROUP=/etc/group
+  USER_ID=$(id -u)
+  GROUP_ID=$(id -g)
+
+  if [ x"$USER_ID" != x"0" -a x"$USER_ID" != x"1001" ]; then
+    NSS_WRAPPER_PASSWD=/opt/openshift/passwd
+    NSS_WRAPPER_GROUP=/etc/group
+
+    grep -v ^postgres /etc/passwd > $NSS_WRAPPER_PASSWD
+    echo "postgres:x:${USER_ID}:${GROUP_ID}:PostgreSQL Server:${HOME}:/bin/bash" >> $NSS_WRAPPER_PASSWD
+
+    export NSS_WRAPPER_PASSWD
+    export NSS_WRAPPER_GROUP
+
+    LD_PRELOAD=libnss_wrapper.so
+    export LD_PRELOAD
```

Comment:
symmetry w/ the others?  :) 

I think this was the result of some rework/reordering. i'll change it.